app/jobs/resolution_proofing_job.rb

```diff
@@ -73,6 +73,16 @@ def perform(
       device_profiling_success: callback_log_data&.device_profiling_success,
       timing: timer.results,
     )
+
+    if IdentityConfig.store.idv_socure_shadow_mode_enabled
+      SocureShadowModeProofingJob.perform_later(
+        document_capture_session_result_id: document_capture_session.result_id,
+        encrypted_arguments:,
+        service_provider_issuer:,
+        user_email: user_email_for_proofing(user),
+        user_uuid: user.uuid,
+      )
```

Comment:
More to do here around including phone number (which we don't currently pass to ResolutionProofingJob). Captured as [LG-14282](https://cm-jira.usa.gov/browse/LG-14282)

Reply:
did we consider enqueuing this job in parallel from inside IDV agent?

Reply:
This job depends on the resolution proofing job completing before it can run (since it reads results from the proofing doc capture session). So parallel wouldn't work here

Reply:
Is that so we can compare the results easily in Cloudwatch? I feel like we would be able to accomplish that with a `coalesce() | stats ... by properties.user_id` or maybe we come up with a job id to join them by that's shared?

Reply:
Ooh, yeah, I like the job id idea, lemme noodle on that

Reply:
There is maybe an argument for using `trace_id` as a value that can connect the two jobs... It gets passed into `ResolutionProofingJob`, but then the `IdV: doc auth verify proofing results` event gets logged by the verify info concern as part of a different request, and so will have a different trace id.

So then I thought maybe create a new `socure_shadow_mode_job_id`, but that gets a little awkward because we have to store it on the DocumentCaptureSession so that the verify info concern can include it in the payload sent to `verify proofing results` so that we can then link those events with the socure events.

I don't think it's good enough to use user_id as a key here, since a single user can have multiple submissions.

Which brings me back to the solution I'm using here: schedule the job after the resolution proofing job has run, log payloads for existing + socure in a new event. 

The intention here is to run this for a short period of time to gather data, so one advantage to this approach is that the change is _mostly_ contained to the `SocureShadowModeProofingJob` class, with a little tweak to `ResolutionProofingJob`.

(I did just create [LG-14328](https://cm-jira.usa.gov/browse/LG-14328) to try addressing some of the awkwardness caused by logging our event from a controller rather than a background job)

Reply:
ok! I was thinking this, but for all the reasons you stated this seems fine as-is too

```ruby
job_id = SecureRandom.uuid
ResolutionProofingJob.perform_later(job_id:, ...)
SocureShadowModeProofingJob.perform_later(job_id:, ...)
```